tools/auction-deploy/Makefile

```diff
@@ -11,7 +11,7 @@ test: install
 	$(VIRTUAL_ENV)/bin/pytest tests
 
 compile: install-requirements
-	$(VIRTUAL_ENV)/bin/deploy-tools compile --evm-version petersburg -d $(CONTRACTS_DIRECTORY)
+	$(VIRTUAL_ENV)/bin/deploy-tools compile --evm-version petersburg --optimize-runs 100 -d $(CONTRACTS_DIRECTORY)
```

Comment:
Please add `--optimize` explicit is better than implicit.

Reply:
:heavy_check_mark: